docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -7,27 +7,47 @@ Diarrheal Diseases
 Disease Description
 -------------------
 
+We follow GBD 2017 and "defined diarrhoeal disease episodes as three
+or more loose stools in a 24-hour period." (p. 88 of
+[GBD-2017-YLD-Capstone-Appendix-1]_).
+
 .. todo::
 
-   Adapt material from GBD capstone and other sources, e.g. [WHO]_,
+   Adapt additional material from GBD capstone and other sources, e.g. [WHO]_,
    [CDC]_, [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
-The relevant ICD codes are...
-
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+According to the [GBD-2017-YLD-Capstone-Appendix-1]_, "There are no
+major modelling updates from GBD 2016," (p. 93) and "self-reported
+prevalence is the reference category" (p. 88).
+
+Regarding the duration of a bout of diarrhea, "the mean duration was
+the duration in days, an average of 4.3 (4.2 4.4)". (p. 89, based on a paper referenced there)
+
+The GBD 2017 adjusted for seasonal variation in diarrheal disease, but
+we have not attempted to include this variation in Vivarium yet. (p. 89)
+
+There is substantial additional effort in GBD to divide diarrhea
+burden into the aetiologies of diarrhea, but we have not included
+aetiologies in this simple model.
+
```

Comment:
the non-fatal model is severity split based.  Here we assume people have the average severity for their age/sex/location/year...